### ISSUE TITLE
Draco - Option to use Sequential instead of EdgeBreaker method?

### ISSUE THREAD

**Issue by SpaceIm** [AnalyticalGraphicsInc/gltf-pipeline#402]
Hi

Is there a way to use sequential encoding method of draco instead of the edgebreaker encoding, to preserve the order ?

Basically, in my triangulated mesh, I need no preserve the face order, and the direction (CCW in my case). It is mandatory in our application, since end users can load their own files representing physical values associated to face id of the original mesh, in order to display gradient colors at the surface, based on these values.

Cheers

**Comment by lilleyse** [comment]
Hi @SpaceIm,

Set `--draco.compressionLevel=0` to use sequential encoding.

I updated the documentation to mention this: https://github.com/AnalyticalGraphicsInc/gltf-pipeline/commit/302a48cea07ba7f3a8a144efb5ccc3449d4395ae

**Comment by SpaceIm** [comment]
Tried to set `--draco.compressionLevel=0`, but I'm pretty sure it didn't use sequential encoding in my case.
I've used : `-d --draco-compression 0 --draco.quantisizePositionBits 20 -s`

The faces order is still not preserved and the generated file is by far too small for sequential encoding I guess (from 55Mo for the original glTF (positions and indices only, 3M triangles) to 1 Mo).
For example, with the javascript API of draco, using encodingSpeed = 0, decodingSpeed = 4, method = MESH_SEQUENTIAL_ENCODING and quantisizePositionBits = 20, it creates a draco file of 33Mo with the indices order preserved.
